docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -174,14 +155,13 @@ cmake \
 -D CMAKE_CXX_COMPILER=clang++ \
 -D CMAKE_Fortran_COMPILER=gfortran \
 -D CMAKE_BUILD_TYPE=Debug \
--D BUILD_SHARED_LIBS=OFF \
+-D BUILD_SHARED_LIBS=ON \
+-D BUILD_PYTHON_BINDINGS=ON \
```

Comment:
Pybindings are on by default now, just FYI

Reply:
Good to know! Since they are on by default, I'll remove that line to streamline the docs